python/composio/cli/triggers.py

```diff
@@ -159,6 +159,15 @@ def _enable_trigger(context: Context, name: str) -> None:
 
     config = {}
     properties = trigger.config.properties or {}
+    # Populate default values for optional fields
+    config.update(
+        {
+            field: field_props.default
+            for field, field_props in properties.items()
+            if field_props.default is not None
```

Comment:
Consider adding validation to ensure that default values don't override required fields. While the current implementation looks correct, adding an explicit check would make the behavior more robust.